net/message/blockHdr.go

```diff
@@ -229,6 +229,10 @@ func GetHeadersFromHash(startHash common.Uint256, stopHash common.Uint256) ([]le
 				return nil, 0, err
```

Comment:
This patch doesn't solve this issues you mentioned above, the "leveldb not found" still will be printed and the DB will be accessed too.